package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "aria-at-harness",
+  "version": "0.0.0",
+  "private": true,
+  "bin": {
+    "aria-at-harness-agent": "./bin/agent"
+  },
+  "devDependencies": {
+    "aria-at": "github:w3c/aria-at",
+    "esbuild": "^0.13.3",
+    "husky": ">=6",
+    "jsdoc": "^3.6.7",
+    "lint-staged": ">=10",
+    "prettier": "^2.4.1"
+  },
+  "dependencies": {
+    "yargs": "^17.2.1"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "homepage": "https://github.com/mzgoddard/aria-at-harness",
+  "scripts": {
+    "build": "npm run build:agent && npm run build:docs",
+    "build:agent": "esbuild ./src/agent --bundle --platform=node --external:yargs --sourcemap=inline --outfile=lib/agent.js",
```

Comment:
That was the impression I got from my first review. What do you think about switching to using Node.js's ECMAScript module implementation?

[The LTS release of Node.js has stable support for ECMASCript modules](https://nodejs.org/dist/latest-v14.x/docs/api/esm.html#esm_introduction), so we could rely on the built-in support rather than using a tool like esbuild to transform the declarations. Doing so would improve the development experience by reducing the number of dependencies, simplifying the workflow, and streamlining debugging. Those are pretty substantial benefits, in my opinion, and I can't think of any drawbacks (at least, not for a greenfield project like this).

Reply:
Does the `package.json` `type` field configuration looks good to you?